release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -3295,3 +3295,22 @@ For more information, see xref:../authentication/bound-service-account-tokens.ad
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-52"]
+=== RHBA-2022:8485 - {product-title} 4.9.52 bug fix update
+
+Issued: 2022-11-22
+
+{product-title} release 4.9.52 is now available. 4.9.52 contains no IBM powerbuild architecture. IBM powerbuild architecture will be included in the next z-stream release. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:8485[RHBA-2022:8485] advisory. There are no RPM packages for this release.
```

Comment:
Recommend changing "IBM powerbuild architecture" to "IBM Power build options", per the IBM documentation.

Reply:
Thanks, James! I have updated the PR to reflect the above changes. :)